hilti/toolchain/src/ast/ast-context.cc

```diff
@@ -682,25 +682,24 @@ void ASTContext::_saveIterationAST(const Plugin& plugin, const std::string& pref
 }
 
 static void _recursiveDependencies(const ASTContext* ctx, declaration::Module* module,
-                                   std::vector<declaration::module::UID>* seen) {
-    if ( std::find(seen->begin(), seen->end(), module->uid()) != seen->end() )
+                                   std::set<declaration::module::UID>* seen) {
+    if ( seen->find(module->uid()) != seen->end() )
```

Comment:
No need to `find`, we do not need the element.

```suggestion
    if ( seen->count(module->uid()) )
```